docker/Dockerfile

```diff
@@ -1,14 +1,19 @@
 FROM alpine:3.16.1
 
-RUN apk add --no-cache gettext curl nginx php8 php8-fpm php8-opcache php8-pdo php8-pdo_sqlite php8-openssl && \
+# To reduce duplication
+ENV PHP_VER=php8
+# For use in entrypoint.sh
+ENV PHPFPM_VER=php-fpm8
```

Comment:
It's nothing I'm going to hold up approval on, but maybe make this just `PHP_VER=8` instead of having to have a variable for both PHP and FPM.

Otherwise, I'm really liking this PR! Great work!

Reply:
My only concern with doing it that way is code readability. 

It'd go from:

`$PHP_VER $PHP_VER-fpm $PHP_VER-opcache $PHP_VER-pdo $PHP_VER-pdo_sqlite $PHP_VER-openssl`

To:

`php$PHP_VER php$PHP_VER-fpm php$PHP_VER-opcache php$PHP_VER-pdo php$PHP_VER-pdo_sqlite php$PHP_VER-openssl`

Using one or the other won't affect the Dockerfile obv, but from someone looking at the code, it may seem messy.

Thoughts?

Reply:
Yeah, I was thinking about that as I was typing it. I think it's probably fine the way it is.